test/L2GovernorMetadata.t.sol

```diff
@@ -6,9 +6,13 @@ import "@openzeppelin/contracts/utils/Strings.sol";
 import {L2GovernorMetadata} from "src/L2GovernorMetadata.sol";
 import {L2GovernorMetadataHarness} from "test/harness/L2GovernorMetadataHarness.sol";
 import {TestConstants} from "test/Constants.sol";
+import {L1BlockMock} from "test/mock/L1BlockMock.sol";
 
 contract L2GovernorMetadataTest is TestConstants {
   L2GovernorMetadataHarness l2GovernorMetadata;
+  L1BlockMock mockL1Block;
+
+  uint64 constant MOCK_L1_BLOCK = 18_442_511;
```

Comment:
This can probably be moved to the Constants file as it is used here and in the L1Block mock

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610